installation/getting-started-with-fluent-bit.md

```diff
@@ -17,7 +17,7 @@ description: The following serves as a guide on how to install/deploy/upgrade Fl
 | Operating System       | Installation Instructions                                                                                                                                                                               |
 | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | CentOS / Red Hat       | [CentOS 7](linux/redhat-centos.md#install-on-redhat-centos), [CentOS 8](linux/redhat-centos.md#install-on-redhat-centos)                                                                                |
-| Ubuntu                 | [Ubuntu 16.04 LTS](linux/ubuntu.md#ubuntu-16-04-lts-xenial-xerus), [Ubuntu 18.04 LTS](linux/ubuntu.md#ubuntu-18-04-lts-bionic-beaver), [Ubuntu 20.04 LTS](linux/ubuntu.md#ubuntu-20-04-lts-focal-fossa) |
+| Ubuntu                 | [Ubuntu 16.04 LTS](linux/ubuntu.md#ubuntu-16-04-lts-xenial-xerus), [Ubuntu 18.04 LTS](linux/ubuntu.md#ubuntu-18-04-lts-bionic-beaver), [Ubuntu 20.04 LTS](linux/ubuntu.md#ubuntu-20-04-lts-focal-fossa),[Ubuntu 22.04 LTS](linux/ubuntu.md#ubuntu-22-04-lts-jammy-jellyfish) |
```

Comment:
Add space after the comma.
Check the link. I don't think it's taking you anywhere on the page (ubuntu.md#ubuntu...)